.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: go
+
+go:
+  - 1.4
+  - tip
+
+env:
+  - GOTAGS=restapi
+  #- GOTAGS=powershell

```

Comment:
Well, technically, the powershell implementation still exists, but yes, I'll remove it.